lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
+                <div class="list-item-content">
+                    <div class="list-item-left-section">
+                        <h3> <%= bookmark.get('breadcrumbTrail') %> </h3>
+                        <p id="bookmark-date-<%= index %>" class="list-item-date"> <%= gettext("Bookmarked on") %> <%= bookmark.get('userFriendlyDate') %> </p>
+                    </div>
+
+                    <p id="bookmark-type-<%= index %>" class="list-item-right-section">
+                        <span aria-hidden="true">View</span>
+                        <i class="icon fa fa-arrow-right" aria-hidden="true"></i>
+                    </p>
+                </div>
+            </a>
+        <% }); %>
+    </div>
+
+<% } else { %>
+
+    <div class="bookmarks-empty" tabindex="0">
+        <div class="bookmarks-empty-header">
+            <i class="icon fa fa-bookmark-o bookmarks-empty-header-icon" aria-hidden="true"></i>
+            It looks like your bookmarks list is empty!
```

Comment:
This is looking great, Muhammad.  I noticed you added the "sr-is-focusable"
div.  If you're seeing a dotted line appear above the bookmarks content
when you click the button, don't be concerned.  That bug was fixed here:
https://openedx.atlassian.net/browse/UX-1946

It's fixed on production.

Mark

On Wed, May 6, 2015 at 10:56 AM, srpearce <notifications@github.com> wrote:

> In lms/templates/bookmarks/bookmarks_results.underscore
> <https://github.com/edx/edx-platform/pull/7896#discussion_r29765863>:
>
> > +
> > +                    <p id="bookmark-type-<%= index %>" class="list-item-right-section">
> > +                        <span aria-hidden="true">View</span>
> > +                        <i class="icon fa fa-arrow-right" aria-hidden="true"></i>
> > +                    </p>
> > +                </div>
> > +            </a>
> > +        <% }); %>
> > +    </div>
> > +
> > +<% } else { %>
> > +
> > +    <div class="bookmarks-empty" tabindex="0">
> > +        <div class="bookmarks-empty-header">
> > +            <i class="icon fa fa-bookmark-o bookmarks-empty-header-icon" aria-hidden="true"></i>
> > +            It looks like your bookmarks list is empty!
>
> You have not bookmarked any course content yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7896/files#r29765863>.
>